utils/get_train_eval_files.py

```diff
@@ -0,0 +1,78 @@
+import os
+import pandas as pd
+from random import shuffle
+
+def get_train_eval_files (data_dir, pattern, train_eval_ratio = 0.8, start_id = 100000, end_id = 190000):
+    # choose those subject
+    subject_ids = [i for i in range(start_id, end_id)]
+    subject_path_id = [str(id) + '_30' for id in subject_ids]
+    subject_paths = [os.path.join(data_dir,subject_folder) for subject_folder in subject_path_id]
+
+    # check the existing subject
+    path_collections = []
+    for subject_path in subject_paths:
+        if os.path.exists(subject_path):
+            path_collections.append(subject_path)
+    #print(path_collections)
+
+    # check the existing pattern image
+    path_collections = [os.path.join(path, 'image') for path in path_collections]
+    #print(path_collections)
+
+    data_path = []
+    for item in path_collections:
+        files = os.listdir(item)
+        for file in files:
+            #print(file)
+            if file.endswith(pattern):
+                data_path.append(os.path.join(item, file))
+                break
+    print('train_paths: ', data_path)
+
+    chose_subjects = [path.split('/')[6] for path in data_path]
+    qualityInfo = pd.read_excel('CombiExcel.xlsx')
```

Comment:
Where do you store combiExcel? We can never put it in the repository! 
Please make a path variable in config which can be handed over here

Reply:
I am sorry that this file is for NAKO dataset rather than NAKO_IQA dataset. I have realized the function in the NAKOInfo class, so I will delete this file.

Reply:
you can leave it here:
Just put an if-else case switch. Please add another variable in the config file: "sPathToExcel"
and the line reads then:
`qualityInfo = pd.read_excel(os.path.join(cfg['sPathToExcel'], 'CombiExcel.xlsx'))`